gce/prepull.yaml

```diff
@@ -14,122 +14,83 @@ spec:
       nodeSelector:
         kubernetes.io/os: windows
       initContainers:
-      - image: e2eteam/busybox:1.29
-        name: busybox1
-        command: ["cmd", "exit"]
-      - image: e2eteam/curl:1803
-        name: curl2
+      - image: e2eteam/agnhost:2.8
+        name: agnhost
         command: ["cmd", "exit"]
-      - image: e2eteam/java:openjdk-8-jre
-        name: java3
+      - image: e2eteam/agnhost:2.6
+        name: agnhostprivate
         command: ["cmd", "exit"]
-      - image: e2eteam/test-webserver:1.0
-        name: test-webserver4
+      - image: e2eteam/apparmor-loader:1.0
+        name: apparmor-loader
```

Comment:
the apparmor-loader image is not for Windows.

Reply:
Thanks, removed.